rego/rules/arm/sql/auditing_retention.rego

```diff
@@ -37,20 +37,24 @@ input_type = "arm"
 
 resource_type = "MULTIPLE"
 
-resources = fugue.resources("Microsoft.Sql/servers/auditingPolicies")
-
-is_invalid(resource) {
-	resource.TODO == "TODO" # FIXME
+servers = fugue.resources("Microsoft.Sql/servers")
+auditing_settings = fugue.resources("Microsoft.Sql/servers/auditingSettings")
+
+valid_server_ids[id] {
+	settings := auditing_settings[_]
+	lower(settings.properties.state) == "enabled"
```

Comment:
It might make sense to remove this check since it's in the other rule. I think it makes it easier to figure out what you need to fix when you see the failures.